docs/savegame_format.md

```diff
@@ -184,3 +184,31 @@ We advise you to call this setting `__mypp_auto_destroy_rivers` in the settings
 Doing it this way ensures that a savegame created by these patch-packs can still safely be loaded by unpatched clients.
 They will simply ignore the field and continue loading the savegame as usual.
 The prefix is strongly advised to avoid conflicts with future-settings in an unpatched client or conflicts with other patch-packs.
+
+## Scripts custom data format
+
+Script chunks (`AIPL` and `GSDT`) use `CH_TABLE` chunk type.
+
+At the end of each record there's an `SLE_FILE_U8` to indicate if there's custom data (1) or not (0).
```

Comment:
As this document defines `uint8`, I would use that over the `SLE_FILE` variant. I think that makes more sense. As it has little to do anymore with `SLE_FILE` ofc.